WeakAuras/GenericTrigger.lua

```diff
@@ -3414,7 +3411,7 @@ do
   end
 
   function WeakAuras.GetDBMStage()
-    return currentStage, currentStageTotal
+    return DBM:GetState()
```

Comment:
I don't see any GetState in my local DBM repo, did you mean GetStage?

Reply:
There is no reason #4483 does not also happen with bigwigs triggers, but their GetStage function is in BossPrototype.lua so i don't know the equivalent to this DBM:GetStage() for them

Reply:
oh right again...

hmm no idea on how to get to that from bigwigs